pulsar-broker/src/main/java/org/apache/pulsar/broker/service/AbstractTopic.java

```diff
@@ -158,6 +158,8 @@ public abstract class AbstractTopic implements Topic, TopicPolicyListener<TopicP
     protected volatile boolean transferring = false;
     private volatile List<PublishRateLimiter> activeRateLimiters;
 
+    protected Policies lastUpdatedNamespacePolicies;
```

Comment:
Oh, sorry. I thought it was able to get from somewhere before.
Pass via the params is better. :)

Reply:
Sure, reverted